server/migrations/20171024102115_users.js

```diff
@@ -0,0 +1,14 @@
+
+exports.up = function(knex, Promise) {
```

Comment:
I thought the linter would pick up on the lack of spaces after `function`. And on the next line.